packages/apps/docs/src/docs/build/authentication/register-an-account.md

```diff
@@ -2,36 +2,39 @@
 title: Register a Kadena SpireKey account
 description:
   Whether you're a user or developer, creating a Kadena SpireKey account is the
-  first step to using simplified WebAuthN authentication and authorization
+  first step to using simplified WebAuthn authentication and authorization
   services.
 menu: Authenticate accounts
 label: Register an account
 order: 2
 layout: full
 ---
 
-# Register a device to create a new Kadena SpireKey account
+# Register an account with Kadena SpireKey
 
 Whether you're a user or developer, creating a Kadena SpireKey account is the
-first step to using simplified WebAuthN authentication and authorization
+first step to using simplified WebAuthn authentication and authorization
```

Comment:
not just simplified WebAuthn authentication, but jus WebAuthn authentication in general.